pkg/tools/topo-generator/reader/reader.go

```diff
@@ -0,0 +1,60 @@
+package reader
+
+import (
+    "io/ioutil"
+    "gopkg.in/yaml.v3"
+	"github.com/onosproject/onos-lib-go/pkg/logging"
```

Comment:
Same thing for imports. There are not adjusted (they should be under each other).